frontend/src/pages/account-settings/components/subscription/subscription.tsx

```diff
@@ -25,6 +25,14 @@ const Subscription: React.FC = () => {
     return '';
   }, [subscriptionEndDate]);
 
+  const isSubscriptionValid = useMemo(() => {
+    if (subscriptionEndDate) {
+      return checkSubscription(new Date(), new Date(subscriptionEndDate));
+    }
+
+    return false;
```

Comment:
```suggestion
    return checkSubscription(new Date(), new Date(subscriptionEndDate));
```

Reply:
in this case if user don`t have subscription subscriptionEndDate will be 'null'